src/components/DataGroups/ImageDataGroup.js

```diff
@@ -8,14 +8,15 @@ export const ImageDataGroup = (column) => {
     field: { alias },
   } = column;
   const root = getRoot(original);
+  const objectFit = "contain";
```

Comment:
Can we add this css property directly in css? That'd be great if we could rip out these inline css.